keps/sig-cli/2368-kubectl-default-container/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: kubectl default container
+kep-number: 2368
+authors:
+  - "@pacoxu"
+owning-sig: sig-cli
+participating-sigs:
+status: provisional
```

Comment:
IMO this is implementable, we can try to get this merged (covering all questions) and move directly to implementable :)